code/modules/uplink/uplink_items.dm

```diff
@@ -1788,6 +1759,15 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 6
 	restricted_roles = list("Geneticist", "Research Director")
 
+/datum/uplink_item/role_restricted/brainwash_disk
+	name = "Brainwashing Surgery Program"
+	desc = "A disk containing the procedure to perform a brainwashing surgery, allowing you to implant an objective onto a target. \
+	Insert into an Operating Console to enable the procedure."
+	item = /obj/item/disk/surgery/brainwashing
+	restricted_roles = list("Medical Doctor", "Chief Medical Officer", "Roboticist")
+	cost = 5
+
+/* Skyrat Edit: Spawns 50 clowns. Skyrat players are too unrobust to handle this.
```

Comment:
somehow I don't think this is a PR we should be taking seriously.

Reply:
The host told me to disable this.